litestar/channels/backends/redis.py

```diff
@@ -23,6 +23,35 @@
 _XADD_EXPIRE_SCRIPT = (_resource_path / "_redis_xadd_expire.lua").read_text()
 
 
+class _LazyEvent:
+    """A lazy proxy to asyncio.Event that only creates the event once it's accessed.
```

Comment:
```suggestion
    """A lazy proxy to ``asyncio.Event`` that only creates the event once it's accessed.
```
or maybe
```rst
asyncio.Event :class:`~asyncio.Event`
```
 or similar

Reply:
oh its private